docker/CMSRucioClient/traces/query_collect.json

```diff
@@ -0,0 +1,29 @@
+{
+    "_source": [
+        "metadata.timestamp",
+        "data.meta_data",
+        "data.LFNArray",
+        "data.LFNArrayRef",
+        "data.steps.input.lfn",
+        "data.steps.site",
+        "data.fallbackFiles"
+    ],
+    "from": 0,
+    "size": 10000,
+    "sort": [
+        {
+            "metadata.timestamp": "asc"
+        }
+    ],
+    "query": {
+        "bool": {
+            "must": [
+                {
+                    "exists": {
+                        "field": "data.steps.input"
```

Comment:
It may be more clear to put the timestamps here already so that one can see the full JSON structure and then just change them in the python code, but I leave that to you.